.github/workflows/test-package.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           sdk: ${{ matrix.sdk }}
       - id: install
         name: Install dependencies
-        run: dart pub get
+        run: dart pub upgrade
```

Comment:
I would prefer we just upgrade our dependencies instead. Using upgrade defeats the lockfile. https://github.com/dart-lang/homebrew-dart/pull/114